feat(cache): add playername cache entry

nwvh opened the pull request:
This PR will probably be useless, as the player's name won't change until he rejoins with a different name.

Usage:

```lua
print("player name: ", cache.playername)
```

Mirrrrrow commented on resource/cache/client.lua:
```diff
@@ -51,6 +52,10 @@ CreateThread(function()
 
 		cache:set('weapon', hasWeapon and currentWeapon or false)
 
+		local playername = GetPlayerName(PlayerId())
```
why don't you directly use the cached playerId instead of PlayerId() and use camelCase

nwvh replied:
you're right, also just realized I don't have to cache the player name in a loop as it doesn't change, I can just cache it once after the script has loaded

Mirrrrrow commented on resource/cache/client.lua:
```diff
@@ -51,6 +52,10 @@ CreateThread(function()
 
+		local playername = GetPlayerName(PlayerId())
+
+		cache:set('playername', playername)
```
here also use camelCase

nwvh replied:
I pushed another commit. should be all good now, thanks for the heads up

Mirrrrrow reviewed (commented):
i would suggest this

nwvh commented:
Updated the commit, now using camelCase and `cache.playerId` instead of `PlayerId()` as @Mirrrrrow suggested. I also modified it so that the playername is not being cached in the loop, but only once, just like the cache.playerId.

[#499 ](https://github.com/overextended/ox_lib/commit/61a7a7e2ce7885bb927f900288cb4029d60df57a)

thelindat commented:
Unnecessary. I probably wouldn't even have bothered with `playerId` if I had written cache today, fwiw.
